package.json

```diff
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^8.0.1",
```

Comment:
Is it needed?

Reply:
Not sure , it seems `npm install` add it

Reply:
can you remove it? It is not necessary.

Reply:
this one is not in my change set